applications/openshift/kubelet/kubelet_enable_protect_kernel_sysctl_kernel_panic_on_oops/rule.yml

```diff
@@ -92,7 +103,8 @@ references:
   srg: SRG-APP-000516-CTR-001325
 
 template:
-  name: lineinfile
-  vars:
-    path: /etc/sysctl.d/90-kubelet.conf
-    text: "kernel.panic_on_oops=1"
+    name: sysctl
```

Comment:
nit: doesn't look like this affects the tests, but the indentation shifted right a layer.